contracts/hackatom/src/contract.rs

```diff
@@ -149,15 +149,12 @@ fn query_verifier<S: Storage, A: Api>(deps: &Extern<S, A>) -> Result<Vec<u8>> {
 #[cfg(test)]
 mod tests {
     use super::*;
-    use cosmwasm::mock::{dependencies, mock_env};
-    use cosmwasm::storage::transactional_deps;
-    // import trait to get access to read
-    use cosmwasm::traits::ReadonlyStorage;
-    use cosmwasm::types::coin;
+    // import trait ReadonlyStorage to get access to read
+    use cosmwasm::{coin, mock_dependencies, mock_env, transactional_deps, ReadonlyStorage};
```

Comment:
This mixes production and testing code on the top level.
I do like pulling things up, but what about exporting all production code under `cosmwasm::*` and all test helpers under `cosmwasm::testing:*` or such. This can also be an inline mod in `lib.rs` that re-exports from other places. But just to make it quite clear if you are doing something wrong.

I also like the rename to `mock_dependencies`

Reply:
This indicates that we should have two crates here: one that is used from within the contract and one for testing. The testing crate does not need to compile to Wasm.

But I agree, separating out the testing symbols is a good first step towards that.